src/types/mysql-message/mysql-error.message.ts

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Dodatkowo - nie uważam, żę to są typy - bardziej bym się skłonił, że to są jakieś wiadomości, więc wrzuciłbym to do `utils/message/mysql-message`. Do przemyślenia przez Ciebie.

Reply:
Rozumiem Twoje podejście. Enum to typy, jednak możemy je traktować jak wiadomości, chociaz wtedy zamiast enum użyłbym messages.constants.ts, jednak zastosuje Twoje rozwiązanie.